packages/enrichPrescriptions/tests/testStatusUpdate.test.ts

```diff
@@ -190,4 +196,130 @@ describe("Unit tests for statusUpdate", function () {
 
     expect(requestBundle).toEqual(expectedResponseBundle)
   })
+
+  it("status update does not replace prescriber cancelled status", async () => {
+    const requestBundle = simpleRequestBundle()
+    const requestCollectionBundle = requestBundle.entry![0].resource as Bundle
+    const medicationRequest = requestCollectionBundle.entry![0].resource as MedicationRequest
+
+    // Add the initial extension for prescription cancelled 30 minutes ago
+    const updateTime = new Date(SYSTEM_DATETIME.valueOf() - 1000 * 60 * 30).toISOString()
+    addExtensionToMedicationRequest(medicationRequest, "Prescriber Cancelled", updateTime)
+
+    const statusUpdates = simpleStatusUpdatesPayload()
+    applyStatusUpdates(requestBundle, statusUpdates)
+
+    // Check that the status has been updated to 'Prescriber Cancelled'
+    expect(medicationRequest.extension![0].extension![0].valueCoding!.code).toEqual("Prescriber Cancelled")
+  })
+
+  describe("Delay WithPharmacy status", () => {
+    it("when the last update is '0002 - With Dispenser', updated less than an hour ago, and missing NPPT updates, set status as 'Prescriber Approved'", async () => {
```

Comment:
The wording of the test descriptions are a bit confusing here. It isn't immediately obvious that '0002 - With Dispenser' maps to 'With Pharmacy but Tracking not Supported'.

I'd suggest something like "when 'With Pharmacy but Tracking not Supported' is returned by PfP, updated less than an hour ago..."

Reply:
13da85e71b843341672d94fdcd2b04700c4d409b